js/libs/form.js

```diff
@@ -176,6 +172,18 @@ Fliplet.Widget.instance('form-builder', function(data) {
               }
               break;
 
+            case (!field._submit && typeof field._submit !== 'undefined' || typeof entry.data[field.name] === 'undefined'):
+              /**
+               * In case if field._submit is not undefined or we received undefined entry.data[field.name]
+               * for example a user forgot to add any value in his code.
+               * In these cases, we do not change a field value
+               * We moved this `typeof entry.data[field.name] === 'undefined' ` condition to this position 
+               * because when we receive an undefined value in the 
+               * date or time input we should set them to the current date or time.
+              */
+              field.value = field.value;
```

Comment:
I don't understand why we have this `case` at all if all it does is assign a variable to itself. It looks like it doesn't do anything at all, thus the `case` is unnecessary?

Reply:
@tonytlwu  
This case was initially here, as far as I understand it I described in the comment above it.
But when I tried to removed this case nothing was changed.
Should I remove it from the code?

Reply:
I agree with the analysis, let's remove this unnecessary block.